build-logic/convention/src/main/kotlin/AndroidFirebaseConventionPlugin.kt

```diff
@@ -30,7 +30,8 @@ class AndroidFirebaseConventionPlugin : Plugin<Project> {
                 "implementation"(platform(libs.findLibrary("firebase.bom").get()))
                 "implementation"(libs.findLibrary("firebase.auth").get())
                 "implementation"(libs.findLibrary("firebase.authUI").get())
-                "implementation"(libs.findLibrary("firebase.dynamicLink").get())
+                // Remove this line if it exists
+                // "implementation"(libs.findLibrary("firebase.dynamicLink").get())
```

Comment:
Removed the Firebase Dynamic Links dependency. This change is necessary as part of the process of integrating Airbridge SDK for deep linking functionality. As mentioned in the Airbridge SDK documentation, when using Airbridge for deep linking, you should remove any existing deep linking solutions to avoid conflicts. This change ensures that the app will now use Airbridge for handling deep links instead of Firebase Dynamic Links.